image/templates/helm/stackrox-secured-cluster/internal/defaults/70-scanner-v4.yaml.htpl

```diff
@@ -0,0 +1,33 @@
+scannerV4:
+  disable: [< not .FeatureFlags.ROX_SCANNER_V4 >]
+  indexer:
+    disable: [< not .FeatureFlags.ROX_SCANNER_V4 >]
+    replicas: 3
+    logLevel: INFO
+    autoscaling:
+      disable: false
+      minReplicas: 2
+      maxReplicas: 5
+    resources:
+      requests:
+        memory: "1500Mi"
+        cpu: "1000m"
+      limits:
+        memory: "4Gi"
+        cpu: "2000m"
+  matcher:
+    disable: [< not .FeatureFlags.ROX_SCANNER_V4 >]
+  db:
+    postgresConfig: "@config-templates/scanner-v4-db/postgresql.conf|config-templates/scanner-v4-db/postgresql.conf.default"
+    hbaConfig: "@config-templates/scanner-v4-db/pg_hba.conf|config-templates/scanner-v4-db/pg_hba.conf.default"
+    resources:
+      limits:
+        cpu: "2000m"
+        memory: "4Gi"
+      requests:
+        cpu: "200m"
+        memory: "200Mi"
+
+scannerV4DBPVCDefaults:
+  claimName: "scanner-v4-db"
+  size: "10Gi"
```

Comment:
I don't fully know why this is separate. I kinda just copied what Central DB did